addons/binding/pom.xml

```diff
@@ -54,6 +54,7 @@
     <module>org.openhab.binding.rfxcom.test</module>
     <module>org.openhab.binding.samsungtv</module>
     <module>org.openhab.binding.smaenergymeter</module>
+    <module>org.openhab.binding.solarlog</module>
```

Comment:
To have the binding being picked up by the distro, you furthermore need to add it to the feature.xml, again at the alphabetically correct position. If you have a dependency on some transport bundle (e.g. upnp, mdns or serial), make sure to add a line for this dependency as well (see the other bindings as an example).